app/models/game.rb

```diff
@@ -34,6 +34,40 @@ def populate_board!
   end
 
   def check?
-    # code
+    white_king = King.find_by(color: 'white')
+    black_king = King.find_by(color: 'black')
```

Comment:
This code is looking at all of the `King`'s in our database.

We only want the `King`s inside of this game.

I think you can do... `pieces.kings.find_by...` (try it in a console to see) 🎉

Reply:
`pieces.kings.find_by` didn't seem to work, because it said `kings` was not a valid method.

However, I changed it to the following:
`white_king = King.find_by(color: 'white', game: self)
black_king = King.find_by(color: 'black', game: self)`

Please let me know if this is okay.